lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -247,14 +247,14 @@ void GUIUtils::displaySensorAverage(int average) {
     }
 #endif
     char output[4];
-    sprintf(output, "%03d", average);
+    sprintf(output, "%04d", average);
     displayCenterBig(output);
 }
 
 // TODO: separate this function, format/display
 void GUIUtils::displaySensorData(int mainValue, int chargeLevel, float humi, float temp, int rssi) {
     char output[22];
-    sprintf(output, "%03d E%02d H%02d%% T%02d°C", mainValue, 0, (int) humi, (int) temp);
+    sprintf(output, "%04d E%02d H%02d%% T%02d°C", mainValue, 0, (int) humi, (int) temp);
```

Comment:
Here too, but maybe we need a extra parameter, something like **String unit**, because maybe in the future the user will change it, the main value, from the Android app, and these are different units, somethink like that:

```cpp
void GUIUtils::displaySensorData(int mainValue, String unit, int chargeLevel, float humi, float temp, int rssi)
```

Reply:
done with deviceType